package.json

```diff
@@ -65,9 +66,8 @@
     "eslint": "^8.38.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-header": "^3.1.1",
-    "eslint-plugin-jsdoc": "^41.1.1",
+    "eslint-plugin-jsdoc": "^50.7.1",
```

Comment:
I bumped up some versions during my attempt to fix the tsdoc issues. I ended up removing tsdoc but the other version bumps seems ok to proceed.